tests/integration_python/test_main_cli.py

```diff
@@ -1440,3 +1440,118 @@ def test_info_output_extended(
     )
 
     assert info_data == snapshot(matcher=path_matcher)
+
+
+def test_feature_name_with_dots_escaping(pixi: Path, tmp_pixi_workspace: Path) -> None:
```

Comment:
I think this test can be condensed a bit, multiple installs can be slow and if its just about the parsing it does the entire toml anyways, so a single `pixi lock` would suffice I think.